src/data/int/modeq.lean

```diff
@@ -64,13 +64,13 @@ begin
 end
 
 theorem modeq.dvd : a ≡ b [ZMOD n] → n ∣ b - a := modeq_iff_dvd.1
-theorem modeq_of_dvd : n ∣ b - a → a ≡ b [ZMOD n] := modeq_iff_dvd.2
+theorem of_dvd : n ∣ b - a → a ≡ b [ZMOD n] := modeq_iff_dvd.2
```

Comment:
You're not in the `modeq` namespace here; did you really intend for this to be `int.of_dvd`?

Reply:
Oh, that's a regex accident. I Ctrl+F ed `modeq.of_dvd`